docs/content/how-to/aws/disaster-recovery.md

```diff
@@ -1,25 +1,80 @@
+
+1. *The control-plane is down for your hosted cluster (api-server, etcd,...)*
+- For the first situation this procedure it's a good fit.
+
+2. *The management cluster API-server/etcd is down?*
+- For this second situation it does not make sense to use this procedure, so you can use a backup/recovery tool like Velero in order to recover the ETCD.
+
+3. *Hypershift operator is not working, it’s down and it cannot be recovered*
+- For this situation the procedure it's a good fit.
+
+!!! note Important
+    Other situations should be carefully dissected, to ensure the stability of the other deployed HostedClusters
+
+!!! warning
+    These are some examples where this procedure could be useful, we don't recommend following this procedure unless it is strictly necessary.
+
+### Preface and Considerations
 
 The behaviour of this implementation is focused on the transparency for the user. Hypershift will not disrupt any customer workloads at anytime, also have in mind that the service workloads will be up and running during the migration process. Maybe at some point the Cluster API will be down but this will not affect the services running on the worker nodes.
 
 In the storage side, It's mandatory to have under consideration that when we move a HostedControlPlane to another Management cluster we use some external services, which allows the migration to make it happen. We re-use the storage provisioned in AWS by the initial ControlPlane (PVs/PVCs) in the destination Management cluster.
 
 Regarding the Workers nodes assigned to the cluster, during the migration they will still point to the same DNS entry, and we, under the hood, change the DNS Records to point to the new Management Cluster API, that way the node migration is transparent for the user.
 
-!!! note
-    One considerations we should have is, both Hypershift deployments Source Management cluster and Destination Management cluster, had to have the `--external-dns` flags, in order to maintain the API server URL with the same entry. If that's not the case, the HostedCluster could not be migrated.
+!!! important
+    Have in mind that this **"migration"** capability, it's only for **disaster recovery purposes**, please **DO NOT** use this to perform clusters migrations as a common task in your platform.
 
-```bash
---external-dns-provider=aws \
---external-dns-credentials=<AWS Credentials location> \
---external-dns-domain-filter=<DNS Base Domain>
-```
+- Hypershift Deployment access endpoints arguments
+
+=== "**Public** and **PublicAndPrivate**"
+    ```bash
+    --external-dns-provider=aws \
+    --external-dns-credentials=<AWS Credentials location> \
+    --external-dns-domain-filter=<External DNS for HostedCluster>
+    ```
+
+=== "**Private**"
+
+    ```bash
+    --private-platform aws \
+    --aws-private-creds <Path to AWS Credentials> \
+    --aws-private-region <AWS Region>
+    ```
+
+- HostedCluster Deployment
```

Comment:
What is "HostedCluster Deployment"?

Reply:
ok, can we clarify that? e.g. "The arguments of the CLI when creating a HostedCluster"